src/imagery/i.eodag/i.eodag.py

```diff
@@ -620,6 +654,91 @@ def products_compare(first, second):
     return search_result
 
 
+def save_footprints(search_result, map_name):
```

Comment:
AFAIU from the eodag workshop, the search results can be saved as geojson. For the footprints vector map, isn't it much easier to get that geojson with all attributes and directly import it into grass?? I believe that is more straightforward and we offer all attr for each footprint, i.e. if the user wants the footprint within grass, just use the save_search_result function below to get a geojson and then v.import to import into GRASS. @lucadelu @ninsbl what do you think?

Reply:
Sounds reasonable to me, so I tend to agree...